ci/scripts/r_test.sh

```diff
@@ -73,7 +73,7 @@ SCRIPT="as_cran <- !identical(tolower(Sys.getenv('NOT_CRAN')), 'true')
     }
     rcmdcheck::rcmdcheck(build_args = '--no-build-vignettes', args = c('--no-manual', '--ignore-vignettes', '--run-donttest'), error_on = 'warning', check_dir = 'check')
   }"
-echo "$SCRIPT" | ${R_BIN} --no-save
+echo "$SCRIPT" | timeout -k 3600 3600 ${R_BIN} --no-save
```

Comment:
I wonder if this is better or if we should add to line 74 the argument `timeout = 3600` which will timeout at the check level (and will mark the build as failed if the timeout happens).

Reply:
Yes, this is much better.  My initial searching on rcmdcheck didn't find any timeout arg otherwise I would have gone this route originally.